src/main/java/org/broadinstitute/hellbender/tools/GatherVcfsCloud.java

```diff
@@ -305,7 +305,7 @@ private static void gatherConventionally(final SAMSequenceDictionary sequenceDic
         try (final VariantContextWriter out = new VariantContextWriterBuilder().setOutputFile(outputFile)
                 .setReferenceDictionary(sequenceDictionary).setOptions(options).build()) {
 
-            final ProgressLogger progress = new ProgressLogger(log, 10000);
+            final ProgressLogger progress = new ProgressLogger(log, 100000);
```

Comment:
As a way of preserving the current behavior and thus making a merge back to master more palatable, this could be an `@Argument` with a default value of 10000. In our WDL where we invoke this we could then explicitly specify 100000 (or whatever value we want this to be).